.env.example

```diff
@@ -12,6 +12,7 @@ REACT_APP_API_BASE_URL=http://localhost/api
 
 # for Hardhat
 ETHEREUM_RPC_URL=https://mainnet.infura.io/v3/YOUR_KEY
+ADMIN_ADDRESS=0x0
```

Comment:
There is a vault owner and a circle admin in the contracts - rename this to be more specific
`HH_OWNER_ADDRESS` for example